---
title: 2021
nav_order: 5
parent: 2025-2021
---

# RECOMB 2021 Proceedings

### Padova, Italy (virtual), August 29 – September 1, 2021  
**PC Chair:** [Jian Peng](https://jianpeng.web.engr.illinois.edu/), University of Illinois at Urbana-Champaign  
**Organization Committee:**  Fabio Vandin (co-chair), Jian Ma (co-chair), Matteo Comin, Barbara Di Camillo, Leonardo Pellegrina, Cinzia Pizzi  
**Keynote Speakers:** Trey Ideker, Ming Li, Katie Pollard, Aviv Regev, Marie-France Sagot, Mona Singh  

# List of Publications


- **Biobank-scale estimation of the proportion of trait variance explained by gene-environment interactions**. Ali Pazokitoroudi, Andy Dahl, Noah Zaitlen, Saharon Rosset, Sriram Sankararaman.
  - Journal: **A scalable and robust variance components method reveals insights into the architecture of gene-environment interactions underlying complex traits**. Ali Pazokitoroudi, Zhengtong Liu, Andrew Dahl, Noah Zaitlen, Saharon Rosset, Sriram Sankararaman. *American Journal of Human Genetics*, 111(7): 1462-1480, 2024.
    - DOI: [10.1016/j.ajhg.2024.05.015](https://doi.org/10.1016/j.ajhg.2024.05.015)


- **Fast Alignment-Free Similarity Estimation By Tensor Sketching**. Amir Joudaki, Gunnar Ratsch, Andre Kahles.
  - Preprint: [bioRxiv 2020.11.13.381814](https://doi.org/10.1007/2020.11.13.381814)


- **Assembling Long Accurate Reads Using de Bruijn Graphs**. Anton Bankevich, Andrey Bzikadze, Mikhail Kolmogorov, Pavel Pevzner.
  - Preprint: [bioRxiv 2020.12.10.420448](https://doi.org/10.1007/2020.12.10.420448)
  - Journal: **Multiplex de Bruijn graphs enable genome assembly from long, high-fidelity reads**. Anton Bankevich, Andrey V. Bzikadze, Mikhail Kolmogorov, Dmitry Antipov, Pavel A. Pevzner. *Nature Biotechnology*, 40: 1075–1081, 2022.
    - DOI: [10.1038/s41587-022-01220-6](https://doi.org/10.1038/s41587-022-01220-6)


- **Freddie: Annotation-independent Detection and Discovery of Transcriptomic Alternative Splicing Isoforms**. Baraa Orabi, Brian McConeghy, Cedric Chauve, Faraz Hach.
  - Preprint: [bioRxiv 2021.01.20.427493](https://doi.org/10.1007/2021.01.20.427493)
  - Journal: **Freddie: annotation-independent detection and discovery of transcriptomic alternative splicing isoforms using long-read sequencing**. Baraa Orabi, Ning Xie, Brian McConeghy, Xuesen Dong, Cedric Chauve, Faraz Hach. *Nucleic Acids Research*, 51(2):e11, 2023.
    - DOI: [10.1093/nar/gkac1112](https://doi.org/10.1093/nar/gkac1112)


- **Minimizer-space de Bruijn Graphs**. Barış Ekim, Bonnie Berger, Rayan Chikhi.
  - Preprint: [bioRxiv 2021.06.09.447586](https://doi.org/10.1007/2021.06.09.447586)
  - Journal: **Minimizer-space de Bruijn graphs: Whole-genome assembly of long reads in minutes on a personal computer**. Barış Ekim, Bonnie Berger, Rayan Chikhi. *Cell Systems*, 12(10):958-968.e6, 2021.
    - DOI: [10.1016/j.cels.2021.08.009](https://doi.org/10.1016/j.cels.2021.08.009)


- **SPICEMIX: Integrative single-cell spatial modeling for inferring cell identity**. Benjamin Chidester, Tianming Zhou, Jian Ma.
  - Preprint: [bioRxiv 2020.11.29.383067](https://doi.org/10.1007/2020.11.29.383067)
  - Journal: **SPICEMIX enables integrative single-cell spatial modeling of cell identity**. Benjamin Chidester, Tianming Zhou, Shahul Alam, Jian Ma. *Nature Genetics*, 55(1):78-88, 2023.
    - DOI: [10.1038/s41588-022-01256-z](https://doi.org/10.1038/s41588-022-01256-z)


- **VeRNAl: Mining RNA Structures for Fuzzy Base Pairing Network Motifs**. Carlos Oliver, Vincent Mallet, Pericles Philippopoulos, William L Hamilton, Jerome Waldispuhl.
  - Preprint: [arXiv:2009.00664](https://doi.org/10.48550/arXiv.2009.00664)
  - Journal: **Vernal: a tool for mining fuzzy network motifs in RNA**. Carlos Oliver, Vincent Mallet, Pericles Philippopoulos, William L Hamilton, Jerome Waldispuhl. *Bioinformatics*, 38(4): 970-976, 2022.
    - DOI: [10.1093/bioinformatics/btab768](https://doi.org/10.1093/bioinformatics/btab768)


- **RepairSig: Deconvolution of DNA damage and repair contributions to the mutational landscape of cancer**. Damian Wójtowicz, Jan Hoinka, Bayarbaatar Amgalan, Yoo-Ah Kim, Teresa M Przytycka.
  - Preprint: [bioRxiv 2020.11.21.392878](https://doi.org/10.1007/2020.11.21.392878)
  - Journal: **RepairSig: Deconvolution of DNA damage and repair contributions to the mutational landscape of cancer**. Damian Wójtowicz, Jan Hoinka, Bayarbaatar Amgalan, Yoo-Ah Kim, Teresa M Przytycka. *Cell Systems*, 12(10):994-1003.e4, 2021.
    - DOI: [10.1016/j.cels.2021.07.004](https://doi.org/10.1016/j.cels.2021.07.004)


- **SPRISS: Approximating Frequent k-mers by Sampling Reads, and Applications**. Diego Santoro, Leonardo Pellegrina, Fabio Vandin.
  - Preprint: [arXiv:2101.07117](https://doi.org/10.48550/arXiv.2101.07117)
  - Journal: **SPRISS: approximating frequent k-mers by sampling reads, and applications**. Diego Santoro, Leonardo Pellegrina, Matteo Comin, Fabio Vandin . *Bioinformatics*, 38(13):3343–3350, 2022.
    - DOI: [10.1093/bioinformatics/btac180](https://doi.org/10.1093/bioinformatics/btac180)


- **Raptor: A fast and space-efficient pre-filter for querying very large collections of nucleotide sequences**. Enrico Seiler, Svenja Mehringer, Mitra Darvish, Etienne Turc, Knut Reinert.
  - Preprint: [bioRxiv 2020.10.08.330985](https://doi.org/10.1007/2020.10.08.330985)
  - Journal: **Raptor: A fast and space-efficient pre-filter for querying very large collections of nucleotide sequences**. Enrico Seiler, Svenja Mehringer, Mitra Darvish, Etienne Turc, Knut Reinert. *iScience*, 24(7):102782, 2021.
    - DOI: [j.isci.2021.102782](https://doi.org/j.isci.2021.102782)


- **DeCiFering the Elusive Cancer Cell Fraction in Tumor Heterogeneity and Evolution**. Gryte Satas, Simone Zaccaria, Mohammed El-Kebir, Benjamin J Raphael.
  - Preprint: [bioRxiv 2021.02.27.429196](https://doi.org/10.1007/2021.02.27.429196)
  - Journal: **DeCiFering the elusive cancer cell fraction in tumor heterogeneity and evolution**. Gryte Satas, Simone Zaccaria, Mohammed El-Kebir, Benjamin J Raphael. *Cell Systems*, 12(10): 1004-1018, 2021.
    - DOI: [10.1016/j.cels.2021.07.006](https://doi.org/10.1016/j.cels.2021.07.006)


- **MoTSE: an interpretable task similarity estimator for small molecular property prediction tasks**. Han Li, Xinyi Zhao, Shuya Li, Fangping Wan, Jianyang Zeng, Dan Zhao.
  - Preprint: [bioRxiv 2021.01.13.426608](https://doi.org/10.1007/2021.01.13.426608)
  - Journal: **Improving molecular property prediction through a task similarity enhanced transfer learning strategy**. Han Li, Xinyi Zhao, Shuya Li, Fangping Wan, Dan Zhao, Jianyang Zeng . *iScience*, 25(10): 105231, 2022.
    - DOI: [10.1016/j.isci.2022.105231](https://doi.org/10.1016/j.isci.2022.105231)


- **An Unrolled Deep Learning Framework for Single Cell Gene Regulatory Networks**. Harsh Shrivastava, Xiuwei Zhang, Srinivas Aluru, Le Song.
  - Preprint: [bioRxiv 2020.04.23.058149](https://doi.org/10.1007/2020.04.23.058149)
  - Journal: **GRNUlar: A deep learning framework for recovering single-cell gene regulatory networks**. Harsh Shrivastava , Xiuwei Zhang, Le Song, Srinivas Aluru. *Journal of Computational Biology*, 29(1):27-44, 2022.
    - DOI: [10.1089/cmb.2021.0437](https://doi.org/10.1089/cmb.2021.0437)


- **Taming Disruptive Base Pairs to Reconcile Positive and Negative Structural Design of RNA**. Hua-Ting Yao, Jerome Waldispuhl, Yann Ponty, Sebastian Will.
  - Preprint: [hal-02987566](https://inria.hal.science/hal-02987566)


- **Study of Real-Valued Distance Prediction For Protein Structure Prediction with Deep Learning**. Jin Li, Jinbo Xu.
  - Preprint: [bioRxiv 2020.11.26.400523](https://doi.org/10.1007/2020.11.26.400523)
  - Journal: **Study of real-valued distance prediction for protein structure prediction with deep learning**. Jin Li, Jinbo Xu. *Bioinformatics*, 37(19):3197–3203, 2021.
    - DOI: [10.1093/bioinformatics/btab333](https://doi.org/10.1093/bioinformatics/btab333)


- **Strain Level Microbial Detection and Quantification with Applications to Single Cell Metagenomics**. Kaiyuan Zhu, Welles Robinson, Alejandro Schaffer, Junyan Xu, Eytan Ruppin, Funda Ergun, Yuzhen Ye, S Cenk Sahinalp.
  - Preprint: [bioRxiv 2020.06.12.149245](https://doi.org/10.1007/2020.06.12.149245)
  - Journal: **Strain level microbial detection and quantification with applications to single cell metagenomics**. Kaiyuan Zhu, Alejandro A. Schäffer, Welles Robinson, Junyan Xu, Eytan Ruppin, A. Funda Ergun, Yuzhen Ye, S. Cenk Sahinalp . *Nature Communications*, 13(1):6430, 2022.
    - DOI: [10.1038/s41467-022-33869-7](https://doi.org/10.1038/s41467-022-33869-7)


- **Re-evaluating deep neural networks for phylogeny estimation: the issue of taxon sampling**. Martin Grosshauser, Paul Zaharias, Tandy Warnow.
  - Journal: **Re-evaluating deep neural networks for phylogeny estimation: the issue of taxon sampling**. Paul Zaharias, Martin Grosshauser, Tandy Warnow . *Journal of Computational Biology*, 29(1):74-89, 2022.
    - DOI: [10.1089/cmb.2021.0383](https://doi.org/10.1089/cmb.2021.0383)


- **Privacy-Preserving Genotype Imputation in a Trusted Execution Environment**. Natnatee Dokmai, Can Kockan, Kaiyuan Zhu, Xiaofeng Wang, S Cenk Sahinalp, Hyunghoon Cho.
  - Preprint: [bioRxiv 2021.02.02.429428](https://doi.org/10.1007/2021.02.02.429428)
  - Journal: **Privacy-preserving genotype imputation in a trusted execution environment**. Natnatee Dokmai, Can Kockan, Kaiyuan Zhu, XiaoFeng Wang, S Cenk Sahinalp, Hyunghoon Cho . *Cell Systems*, 12(10):983-993.e7, 2021.
    - DOI: [10.1016/j.cels.2021.08.001](https://doi.org/10.1016/j.cels.2021.08.001)


- **ScalpelSig: Automated Design of Genomic Panels to Expand Clinical Access to Mutational Signature Analysis**. Nicholas Franzese, Jason Fan, Roded Sharan, Mark DM Leiserson.
  - Journal: **ScalpelSig: Automated Design of Genomic Panels to Expand Clinical Access to Mutational Signature Analysis**. Nicholas Franzese, Jason Fan, Roded Sharan, Mark D M Leiserson. *Journal of Computational Biology*, 29(1): 56-73, 2022.
    - DOI: [10.1089/cmb.2021.0453](https://doi.org/10.1089/cmb.2021.0453)


- **Gromov-Wasserstein optimal transport to align single-cell multi-omics data**. Pinar Demetci, Rebecca Santorella, Bjorn Sandstede, William Stafford Noble, Ritambhara Singh.
  - Preprint: [bioRxiv 2020.04.28.066787](https://doi.org/10.1007/2020.04.28.066787)
  - Journal: **Gromov-Wasserstein optimal transport to align single-cell multi-omics data**. Pinar Demetci, Rebecca Santorella, Björn Sandstede, William Stafford Noble, Ritambhara Singh. *Journal of Computational Biology*, 29(1): 3-18, 2021.
    - DOI: [10.1089/cmb.2021.0446](https://doi.org/10.1089/cmb.2021.0446)


- **Deep unfolding for non-negative matrix factorization with application to mutational signature analysis**. Rami Nasser, Yonina Eldar, Roded Sharan.
  - Preprint: [arXiv:2108.09138](https://doi.org/10.48550/arXiv.2108.09138)
  - Journal: **Deep Unfolding for Non-Negative Matrix Factorization with Application to Mutational Signature Analysis**. Rami Nasser, Yonina C. Eldar, Roded Sharan. *Journal of Computational Biology*, 29(1):45-55, 2022.
    - DOI: [10.1089/cmb.2021.0438](https://doi.org/10.1089/cmb.2021.0438)


- **Alignment and Integration of Spatial Transcriptomics Data**. Ron Zeira, Max Land, Ben Raphael.
  - Preprint: [bioRxiv 2021.03.16.435604](https://doi.org/10.1007/2021.03.16.435604)
  - Journal: **Alignment and integration of spatial transcriptomics data**. Ron Zeira, Max Land, Alexander Strzalkowski, Benjamin J Raphael. *Nature Methods*, 19(5):567-575, 2022.
    - DOI: [10.1038/s41592-022-01459-6](https://doi.org/10.1038/s41592-022-01459-6)


- **Towards the prediction of higher-order genetic interactions**. Ruochi Zhang, Jianzhu Ma, Jian Ma.
  - Preprint: [bioRxiv 2020.11.26.400739](https://doi.org/10.1007/2020.11.26.400739)
  - Journal: **Towards the prediction of higher-order genetic interactions**. Ruochi Zhang, Jianzhu Ma, Jian Ma. *Cell Systems*, to appear, 2025.
    - DOI: [10.1016](https://doi.org/10.1016)


- **Sequence-based prediction of protein-protein interactions: a structure-aware interpretable deep learning model**. Samuel Sledzieski, Rohit Singh, Lenore Cowen, Bonnie Berger.
  - Preprint: [bioRxiv 2021.01.22.427866](https://doi.org/10.1007/2021.01.22.427866)
  - Journal: **D-SCRIPT translates genome to phenome with sequence-based, structure-aware, genome-scale predictions of protein-protein interactions**. Samuel Sledzieski, Rohit Singh, Lenore Cowen, Bonnie Berger. *Cell Systems*, 12(10):969-982.e6, 2021.
    - DOI: [10.1016/j.cels.2021.08.010](https://doi.org/10.1016/j.cels.2021.08.010)


- **Robust Inference of Kinase Activity Using Functional Networks**. Serhan Yılmaz, Marzieh Ayati, Daniela Schlatzer, A Ercument Cicek, Mark Chance, Mehmet Koyuturk.
  - Preprint: [bioRxiv 2020.05.01.062802](https://doi.org/10.1007/2020.05.01.062802)
  - Journal: **Robust inference of kinase activity using functional networks**. Serhan Yılmaz, Marzieh Ayati, Daniela Schlatzer, A Ercument Cicek, Mark Chance, Mehmet Koyuturk. *Nature Communications*, 12(1):1177, 2021.
    - DOI: [10.1038/s41467-021-21211-6](https://doi.org/10.1038/s41467-021-21211-6)


- **Estimating repeat spectra and genome length from low-coverage genome skims with RESPECT**. Shahab Sarmashghi, Metin Balaban, Eleonora Rachtman, Behrouz Touri, Siavash Mirarab, Vineet Bafna.
  - Preprint: [bioRxiv 2021.01.28.428636](https://doi.org/10.1007/2021.01.28.428636)
  - Journal: **Estimating repeat spectra and genome length from low-coverage genome skims with RESPECT**. Shahab Sarmashghi, Metin Balaban, Eleonora Rachtman, Behrouz Touri, Siavash Mirarab, Vineet Bafna. *PLOS Computational Biology*, 17(11):e1009449, 2021.
    - DOI: [10.1371/journal.pcbi.1009449](https://doi.org/10.1371/journal.pcbi.1009449)


- **FaNDOM: Fast Nested Distance-based seeding of Optical Maps**. Siavash Raeisi Dehkordi, Jens Luebeck, Vineet Bafna.
  - Journal: **FaNDOM: Fast nested distance-based seeding of optical maps**. Siavash Raeisi Dehkordi, Jens Luebeck, Vineet Bafna. *Patterns*, 2(5):100248, 2021.
    - DOI: [10.1016/j.patter.2021.100248](https://doi.org/10.1016/j.patter.2021.100248)


- **scDesign2: an interpretable simulator that generates high-fidelity single-cell gene expression count data with gene correlations captured**. Tianyi Sun, Dongyuan Song, Wei Vivian Li, Jingyi Jessica Li.
  - Preprint: [bioRxiv 2020.11.17.387795](https://doi.org/10.1007/2020.11.17.387795)
  - Journal: **scDesign2: a transparent simulator that generates high-fidelity single-cell gene expression count data with gene correlations captured**. Tianyi Sun, Dongyuan Song, Wei Vivian Li, Jingyi Jessica Li. *Genome Biology*, 22: 163, 2021.
    - DOI: [10.1186/s13059-021-02367-2](https://doi.org/10.1186/s13059-021-02367-2)


- **scLink: Inferring Sparse Gene Co-expression Networks from Single-cell Expression Data**. Wei Vivian Li, Yanzeng Li.
  - Preprint: [bioRxiv 2020.09.19.304956](https://doi.org/10.1007/2020.09.19.304956)
  - Journal: **scLink: Inferring Sparse Gene Co-Expression Networks from Single-Cell Expression Data**. Wei Vivian Li, Yanzeng Li . *Genomics, Proteomics & Bioinformatics*, 19(3):475–492, 2021.
    - DOI: [10.1016/j.gpb.2020.11.006](https://doi.org/10.1016/j.gpb.2020.11.006)


- **scAdapt: Virtual adversarial domain adaptation network for single cell RNA-seq data classification across platforms and species**. Xiang Zhou, Hua Chai, Yuansong Zeng, Huiying Zhao, Ching-Hsing Luo, Yuedong Yang.
  - Preprint: [bioRxiv 2021.01.18.427083](https://doi.org/10.1007/2021.01.18.427083)
  - Journal: **scAdapt: Virtual adversarial domain adaptation network for single cell RNA-seq data classification across platforms and species**. Xiang Zhou, Hua Chai, Yuansong Zeng, Huiying Zhao, Yuedong Yang. *Briefings in Bioinformatics*, 22(6):bbab281, 2021.
    - DOI: [10.1093/bib/bbab281](https://doi.org/10.1093/bib/bbab281)


- **Variational Inference Using Approximate Likelihood Under the Coalescent With Recombination**. Xinhao Liu, Huw A Ogilvie, Luay Nakhleh.
  - Preprint: [bioRxiv 2020.08.19.258137](https://doi.org/10.1007/2020.08.19.258137)
  - Journal: **Variational inference using approximate likelihood under the coalescent with recombination**. Xinhao Liu, Huw A Ogilvie, Luay Nakhleh. *Genome Research*, 31(11): 2107-2119, 2021.
    - DOI: [10.1101/gr.273631.120](https://doi.org/10.1101/gr.273631.120)


- **DrugOrchestra: Jointly predicting drug response, targets, and side effects via deep multi-task learning**. Yuepeng Jiang, Stefano Rensi, Sheng Wang, Russ Altman.
  - Preprint: [bioRxiv 2020.11.17.385757](https://doi.org/10.1007/2020.11.17.385757)


- **DEPP: Deep Learning Enables Extending Species Trees using Single Genes**. Yueyu Jiang, Metin Balaban, Siavash Mirarab.
  - Preprint: [bioRxiv 2021.01.22.427808](https://doi.org/10.1007/2021.01.22.427808)
  - Journal: **DEPP: Deep Learning Enables Extending Species Trees using Single Genes**. Yueyu Jiang, Metin Balaban, Qiyun Zhu, Siavash Mirarab. *Systematic Biology*, 72(1):17, 2023.
    - DOI: [10.1093/sysbio/syac031](https://doi.org/10.1093/sysbio/syac031)


- **ProALIGN: directly learning alignments for protein structure prediction via exploiting context-specific alignment motifs**. Lupeng Kong, Fusong Ju, Wei-mou Zheng, Jianwei Zhu, Shiwei Sun, Jinbo Xu,, Dongbo Bu.
  - Preprint: [bioRxiv 2020.12.28.424539](https://doi.org/10.1007/2020.12.28.424539)
  - Journal: **ProALIGN: Directly Learning Alignments for Protein Structure Prediction via Exploiting Context-Specific Alignment Motifs**. Lupeng Kong, Fusong Ju, Wei-mou Zheng, Jianwei Zhu, Shiwei Sun, Jinbo Xu, Dongbo Bu . *Journal of Computational Biology*, 29(2): 92-105, 2022.
    - DOI: [10.1089/cmb.2021.0430](https://doi.org/10.1089/cmb.2021.0430)


- **HiCzin: Normalizing metagenomic Hi-C data and detecting spurious contacts using zero-inflated negative binomial regression**. Yuxuan Du, Sarah M. Laperriere, Jed Fuhrman,, Fengzhu Sun.
  - Preprint: [bioRxiv 2021.03.01.433489](https://doi.org/10.1007/2021.03.01.433489)
  - Journal: **Normalizing Metagenomic Hi-C Data and Detecting Spurious Contacts Using Zero-Inflated Negative Binomial Regression**. Yuxuan Du, Sarah M. Laperriere, Jed Fuhrman, Fengzhu Sun. *Journal of Computational Biology*, 29(2): 106-120, 2022.
    - DOI: [10.1089/cmb.2021.0439](https://doi.org/10.1089/cmb.2021.0439)


- **Deriving Ranges of Optimal Estimated Transcript Expression due to Non-identifiability**. Hongyu Zheng, Cong Ma,, Carl Kingsford.
  - Preprint: [bioRxiv 2019.12.13.875625](https://doi.org/10.1007/2019.12.13.875625)
  - Journal: **Deriving Ranges of Optimal Estimated Transcript Expression due to Nonidentifiability**. Hongyu Zheng, Cong Ma, Carl Kingsford. *Journal of Computational Biology*, 29(2): 121-139, 2022.
    - DOI: [10.1089/cmb.2021.0444](https://doi.org/10.1089/cmb.2021.0444)


- **Set-Min sketch: a probabilistic map for power-law distributions with application to k-mer annotation**. Yoshihiro Shibuya, Djamal Belazzougui,, Gregory Kucherov.
  - Preprint: [bioRxiv 2020.11.14.382713](https://doi.org/10.1007/2020.11.14.382713)
  - Journal: **Set-Min Sketch: A Probabilistic Map for Power-Law Distributions with Application to k-Mer Annotation**. Yoshihiro Shibuya, Djamal Belazzougui, Gregory Kucherov. *Journal of Computational Biology*, 29(2): 140-154, 2022.
    - DOI: [10.1089/cmb.2021.0429](https://doi.org/10.1089/cmb.2021.0429)


- **The statistics of k-mers from a sequence undergoing a simple mutation process without spurious matches**. Antonio Blanca, Robert S. Harris, David Koslicki,, Paul Medvedev.
  - Preprint: [bioRxiv 2021.01.15.426881](https://doi.org/10.1007/2021.01.15.426881)
  - Journal: **The Statistics of k-mers from a Sequence Undergoing a Simple Mutation Process Without Spurious Matches**. Antonio Blanca, Robert S. Harris, David Koslicki, Paul Medvedev. *Journal of Computational Biology*, 29(2): 155-168, 2022.
    - DOI: [10.1089/cmb.2021.0431](https://doi.org/10.1089/cmb.2021.0431)


- **MONI: A Pangenomics Index for Finding MEMs **. Massimiliano Rossi, Marco Oliva, Ben Langmead, Travis Gagie,, Christina Boucher.
  - Preprint: [bioRxiv 2021.07.06.451246](https://doi.org/10.1007/2021.07.06.451246)
  - Journal: **MONI: A Pangenomic Index for Finding Maximal Exact Matches**. Massimiliano Rossi, Marco Oliva, Ben Langmead, Travis Gagie, Christina Boucher. *Journal of Computational Biology*, 29(2): 169-187, 2022.
    - DOI: [10.1089/cmb.2021.0290](https://doi.org/10.1089/cmb.2021.0290)


- **Practical probabilistic and graphical formulations of long-read polyploid haplotype phasing**. Jim Shaw, Yun William Yu.
  - Preprint: [bioRxiv 2020.11.06.371799](https://doi.org/10.1007/2020.11.06.371799)
  - Journal: **flopp: Extremely Fast Long-Read Polyploid Haplotype Phasing by Uniform Tree Partitioning**. Jim Shaw, Yun William Yu. *Journal of Computational Biology*, 29(2): 195-211, 2022.
    - DOI: [10.1089/cmb.2021.0436](https://doi.org/10.1089/cmb.2021.0436)


