---
title: 2022
nav_order: 4
parent: 2025-2021
---

# RECOMB 2022 Proceedings

### San Diego, CA, USA, May 22–25, 2022.  
**PC Chair:** [Itsik Pe’er](https://www.engineering.columbia.edu/faculty-staff/directory/itsik-peer), Columbia University    
**Organization Committee:** Vineet Bafna (chair), Vikas Bansal, Jocelyn Bernardo, Melissa Gymrek, Siavash Mirarab, Glenn Tesler, Kaiyuan Zhu  
**Keynote Speakers:** Regina Barzilay, Howard Y. Chang, John Chodera, Lenore Cowen, John Marioni, Bing Ren, Wenyi Wang  

## List of Publications

- **Unsupervised integration of single-cell multi-omics datasets with disparities in cell-type representation**. Pinar Demetci, Rebecca Santorella, Bjorn Sandstede, Ritambhara Singh.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 3-19, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_1](https://doi.org/10.1007/978-3-031-04749-7_1)
  - Preprint: [bioRxiv 2021.11.09.467903](https://doi.org/10.1101/2021.11.09.467903)
  - Journal: **SCOTv2: Single-Cell Multiomic Alignment with Disproportionate Cell-Type Representation**. Pinar Demetci, Rebecca Santorella, Manav Chakravarthy, Bjorn Sandstede, Ritambhara Singh. *Journal of Computational Biology*, 29(11): 1213-1228, 2022.
    - DOI: [10.1089/cmb.2022.0270](https://doi.org/10.1089/cmb.2022.0270)


- **Semi-supervised single-cell cross-modality translation using Polarbear**. Ran Zhang, William Noble, Laetitia Meng-Papaxanthos, Jean-Philippe Vert.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 20-35, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_2](https://doi.org/10.1007/978-3-031-04749-7_2)
  - Preprint: [bioRxiv 2021.11.18.467517](https://doi.org/10.1101/2021.11.18.467517)
  - Journal: **Multimodal Single-Cell Translation and Alignment with Semi-Supervised Learning**. Ran Zhang, Laetitia Meng-papaxanthos, Jean-philippe Vert, William Stafford Noble. *Journal of Computational Biology*, 29(11): 1198-1212, 2022.
    - DOI: [10.1089/cmb.2022.0264](https://doi.org/10.1089/cmb.2022.0264)


- **Transcription Factor-Centric Approach to Identify Non-Recurring Putative Regulatory Drivers in Cancer**. Jingkang Zhao, Vincentius Martin, Raluca Gordan.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 36-51, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_3](https://doi.org/10.1007/978-3-031-04749-7_3)
  - Preprint: [bioRxiv 2022.01.31.478493](https://doi.org/10.1101/2022.01.31.478493)


- **DeepMinimizer: A Differentiable Framework for Optimizing Sequence-Specific Minimizer Schemes**. Quang Minh Hoang, Hongyu Zheng, Carl Kingsford.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 52-69, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_4](https://doi.org/10.1007/978-3-031-04749-7_4)
  - Preprint: [bioRxiv 2022.02.17.480870](https://doi.org/10.1101/2022.02.17.480870)
  - Journal: **Differentiable Learning of Sequence-Specific Minimizer Schemes with DeepMinimizer**. Quang Minh Hoang, Hongyu Zheng, Carl Kingsford. *Journal of Computational Biology*, 29(12): 1288-1304, 2022.
    - DOI: [10.1089/cmb.2022.0275](https://doi.org/10.1089/cmb.2022.0275)


-**MetaCoAG: Binning Metagenomic Contigs via Composition, Coverage and Assembly Graphs**. Vijini Mallawaarachchi, Yu Lin.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 70-85, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_5](https://doi.org/10.1007/978-3-031-04749-7_5)
  - Preprint: [bioRxiv 2021.09.10.459728](https://doi.org/10.1101/2021.09.10.459728)
  - Journal:**Accurate Binning of Metagenomic Contigs Using Composition, Coverage, and Assembly Graphs**. Vijini Mallawaarachchi, Yu Lin. *Journal of Computational Biology*, 29(12): 1357-1376, 2022.
    - DOI: [10.1089/cmb.2022.0262](https://doi.org/10.1089/cmb.2022.0262)


- **A Fast, Provably Accurate Approximation Algorithm for Sparse Principal Component Analysis Reveals Human Genetic Variation Across the World**. Agniva Chowdhury, Aritra Bose, Samson Zhou, David P Woodruff, Petros Drineas.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 86-106, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_6](https://doi.org/10.1007/978-3-031-04749-7_6)


- **Gene Set Priorization Guided by Regulatory Networks with p-values through Kernel Mixed Model**. Haohan Wang, Oscar Lopez, Wei Wu, Eric Xing.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 107-125, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_7](https://doi.org/10.1007/978-3-031-04749-7_7)
  - Journal: **Kernel Mixed Model for Transcriptome Association Study**. Haohan Wang, Oscar Lopez, Eric P. Xing, Wei Wu. *Journal of Computational Biology*, 29(12): 1353-1356, 2022.
    - DOI: [10.1089/cmb.2022.0280](https://doi.org/10.1089/cmb.2022.0280)


- **Real-valued Group Testing for Quantitative Molecular Assays**. Seyran Saeedi, Myrna Serrano, Dennis G. Yang, J. Paul Brooks, Gregory A. Buck, Tom Arodz.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 126-142, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_8](https://doi.org/10.1007/978-3-031-04749-7_8)
  - Journal: **Group Testing Matrix Design for PCR Screening with Real-Valued Measurements**. Seyran Saeedi, Myrna Serrano, Dennis G. Yang, J. Paul Brooks, Gregory A. Buck, Tomasz Arodz. *Journal of Computational Biology*, 29(12): 1397-1411, 2022.
    - DOI: [10.1089/cmb.2022.0413](https://doi.org/10.1089/cmb.2022.0413)


- **On the Effect of Intralocus Recombination on Triplet-Based Species Tree Estimation**. Max Hill, Sebastien Roch.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 143-158, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_9](https://doi.org/10.1007/978-3-031-04749-7_9)
  - Preprint: [bioRxiv 2021.11.06.467557](https://doi.org/10.1101/2021.11.06.467557)
  - Journal: **Inconsistency of Triplet-Based and Quartet-Based Species Tree Estimation under Intralocus Recombination 8**. Max Hill, Sebastien Roch. *Journal of Computational Biology*, 29(11): 1173-1197, 2022.
    - DOI: [10.1089/cmb.2022.0265](https://doi.org/10.1089/cmb.2022.0265)


- **QT-GILD: Quartet Based Gene Tree Imputation Using Deep Learning Improves Phylogenomic Analyses Despite Missing Data**. Sazan Mahbub, Shashata Sawmya, Arpita Saha, Rezwana Reaz, M Sohel Rahman, Md Shamsuzzoha Bayzid.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 159-176, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_10](https://doi.org/10.1007/978-3-031-04749-7_10)
  - Preprint: [bioRxiv 2021.11.03.467204](https://doi.org/10.1101/2021.11.03.467204)
  - Journal: **Quartet Based Gene Tree Imputation Using Deep Learning Improves Phylogenomic Analyses Despite Missing Data**. Sazan Mahbub, Shashata Sawmya, Arpita Saha, Rezwana Reaz, M Sohel Rahman, Md Shamsuzzoha Bayzid. *Journal of Computational Biology*, 29(11): 1156-1172, 2022.
    - DOI: [10.1089/cmb.2022.0212](https://doi.org/10.1089/cmb.2022.0212)


- **Safe and Complete Flow Decompositions for RNA Assembly**. Shahbaz Khan, Milla Kortelainen, Manuel Caceres, Lucia Williams, Alexandru I Tomescu.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 177-192, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_11](https://doi.org/10.1007/978-3-031-04749-7_11)
  - Preprint: [arXiv:2201.10372](https://doi.org/10.48550/arXiv.2201.10372)
  - Journal: **Improving RNA Assembly via Safety and Completeness in Flow Decompositions**. Shahbaz Khan, Milla Kortelainen, Manuel Caceres, Lucia Williams, Alexandru I Tomescu. *Journal of Computational Biology*, 29(12): 1270-1287, 2022.
    - DOI: [10.1089/cmb.2022.0261](https://doi.org/10.1089/cmb.2022.0261)


- **NetMix2: Unifying network propagation and altered subnetworks**. Uthsav Chitra, Tae Yoon Park, Ben Raphael.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 193-208, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_12](https://doi.org/10.1007/978-3-031-04749-7_12)
  - Preprint: [bioRxiv 2022.01.31.478575](https://doi.org/10.1101/2022.01.31.478575)
  - Journal: **NetMix2: A Principled Network Propagation Algorithm for Identifying Altered Subnetworks**. Uthsav Chitra, Tae Yoon Park, Benjamin J. Raphael. *Journal of Computational Biology*, 29(12): 1305-1323, 2022.
    - DOI: [10.1089/cmb.2022.0336](https://doi.org/10.1089/cmb.2022.0336)


- **Multi-Modal Genotype and Phenotype Mutual Learning to Single-Modal Input for Longitudinal Outcome Prediction**. Alireza Ganjdanesh, Jipeng Zhang, Wei Chen, Heng Huang.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 209-229, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_13](https://doi.org/10.1007/978-3-031-04749-7_13)
  - Journal: **Multimodal Genotype and Phenotype Data Integration to Improve Partial Data-Based Longitudinal Prediction**. Alireza Ganjdanesh, Jipeng Zhang, Sarah Yan, Wei Chen, Heng Huang. *Journal of Computational Biology*, 29(12): 1324-1345, 2022.
    - DOI: [10.1089/cmb.2022.0378](https://doi.org/10.1089/cmb.2022.0378)


- **Fast, Flexible, and Exact Minimum Flow Decompositions via ILP**. Fernando H C Dias, Lucia Williams, Brendan Mumey, Alexandru I Tomescu.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 230-245, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_14](https://doi.org/10.1007/978-3-031-04749-7_14)
  - Preprint: [arXiv:2201.10923](https://doi.org/10.48550/arXiv.2201.10923)
  - Journal: **Efficient Minimum Flow Decomposition via Integer Linear Programming**. Fernando H C Dias, Lucia Williams, Brendan Mumey, Alexandru I Tomescu. *Journal of Computational Biology*, 29(11): 1252-1267, 2022.
    - DOI: [10.1089/cmb.2022.0257](https://doi.org/10.1089/cmb.2022.0257)


- **Co-linear chaining with overlaps and gap costs**. Chirag Jain, Daniel Gibney, Sharma V Thankachan.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 246-262, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_15](https://doi.org/10.1007/978-3-031-04749-7_15)
  - Preprint: [bioRxiv 2021.02.03.429492](https://doi.org/10.1101/2021.02.03.429492)
  - Journal: **Algorithms for Colinear Chaining with Overlaps and Gap Costs**. Chirag Jain, Daniel Gibney, Sharma V Thankachan. *Journal of Computational Biology*, 29(11): 1237-1251, 2022.
    - DOI: [10.1089/cmb.2022.0266](https://doi.org/10.1089/cmb.2022.0266)


- **The Complexity of Approximate Pattern Matching on De Bruijn Graphs**. Daniel Gibney, Sharma V Thankachan, Srinivas Aluru.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 263-278, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_16](https://doi.org/10.1007/978-3-031-04749-7_16)
  - Preprint: [arXiv:2201.12454](https://doi.org/10.48550/arXiv.2201.12454)
  - Journal: **On the hardness of sequence alignment on de Bruijn graphs**. Daniel Gibney, Sharma V Thankachan, Srinivas Aluru. *Journal of Computational Biology*, 29(12): 1377-1396 , 2022.
    - DOI: [10.1089/cmb.2022.0411](https://doi.org/10.1089/cmb.2022.0411)


- **ProTranslator: zero-shot protein function prediction using textual description**. Hanwen Xu, Sheng Wang.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 279-294, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_17](https://doi.org/10.1007/978-3-031-04749-7_17)
  - Preprint: [arXiv:2204.10286](https://doi.org/10.48550/arXiv.2204.10286)


- **Single-cell multi-omic velocity infers dynamic and decoupled gene regulation**. Chen Li, Maria Virgilio, Kathleen Collins, Joshua Welch.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 297-299, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_18](https://doi.org/10.1007/978-3-031-04749-7_18)
  - Preprint: [bioRxiv 2021.12.13.472472](https://doi.org/10.1101/2021.12.13.472472)
  - Journal: **Multi-omic single-cell velocity models epigenome-transcriptome interactions and improves cell fate prediction**. Chen Li, Maria C. Virgilio, Kathleen L. Collins, Joshua D. Welch. *Nature Biotechnology*, 41(3):387-398, 2023.
    - DOI: [10.1038/s41587-022-01476-y](https://doi.org/10.1038/s41587-022-01476-y)


- **Ultrafast and Interpretable Single-cell 3D Genome Analysis with Fast-Higashi**. Ruochi Zhang, Tianming Zhou, Jian Ma.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 300-301, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_19](https://doi.org/10.1007/978-3-031-04749-7_19)
  - Preprint: [bioRxiv 2022.04.18.488683](https://doi.org/10.1101/2022.04.18.488683)
  - Journal: **Ultrafast and interpretable single-cell 3D genome analysis with Fast-Higashi**. Ruochi Zhang, Tianming Zhou, Jian Ma. *Cell Systems*, 13(10): 798-807.e6, 2022.
    - DOI: [10.1016/j.cels.2022.09.004](https://doi.org/10.1016/j.cels.2022.09.004)


- **DiffDomain enables identification of structurally reorganized topologically associating domains**. Dunming Hua, Ming Gu, Yanyi Du, Li Qi, Xiangjun Du, Zhidong Bai, Xiaopeng Zhu, Dechao Tian.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 302-303, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_20](https://doi.org/10.1007/978-3-031-04749-7_20)
  - Preprint: [bioRxiv 2022.12.05.519135](https://doi.org/10.1101/2022.12.05.519135)
  - Journal: **DiffDomain enables identification of structurally reorganized topologically associating domains**. Dunming Hua, Ming Gu, Xiao Zhang, Yanyi Du, Hangcheng Xie, Li Qi, Xiangjun Du, Zhidong Bai, Xiaopeng Zhu, Dechao Tian. *Nature Communications *, 15: 502, 2024.
    - DOI: [10.1038/s41467-024-44782-6](https://doi.org/10.1038/s41467-024-44782-6)


- **Joint inference of repeated evolutionary trajectories and patterns of clonal exclusivity or co-occurrence from tumor mutation trees**. Xiang Ge Luo, Jack Kuipers, Niko Beerenwinkel.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 304-305, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_21](https://doi.org/10.1007/978-3-031-04749-7_21)
  - Preprint: [bioRxiv 2021.11.04.467347](https://doi.org/10.1101/2021.11.04.467347)
  - Journal: **Joint inference of repeated evolutionary trajectories and patterns of clonal exclusivity or co-occurrence from tumor mutation trees**. Xiang Ge Luo, Jack Kuipers, Niko Beerenwinkel. *Nature Communications *, 14: 3676, 2023.
    - DOI: [10.1038/s41467-023-39400-w](https://doi.org/10.1038/s41467-023-39400-w)


- **Fast and Optimal Sequence-to-Graph Alignment Guided by Seeds**. Pesho Ivanov, Benjamin Bichsel, Martin Vechev.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 306-325, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_22](https://doi.org/10.1007/978-3-031-04749-7_22)
  - Preprint: [bioRxiv 2021.11.05.467453](https://doi.org/10.1101/2021.11.05.467453)


- **CLMB: deep contrastive learning for robust metagenomic binning**. Pengfei Zhang, Zhengyuan Jiang, Yixuan Wang, Yu Li.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 326-348, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_23](https://doi.org/10.1007/978-3-031-04749-7_23)
  - Preprint: [bioRxiv 2021.11.15.468566](https://doi.org/10.1101/2021.11.15.468566)


- **Unsupervised cell functional annotation for single-cell RNA-Seq**. Dongshunyi Li, Jun Ding, Ziv Bar-Joseph.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 349-352, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_24](https://doi.org/10.1007/978-3-031-04749-7_24)
  - Preprint: [bioRxiv 2021.11.20.469410](https://doi.org/10.1101/2021.11.20.469410)
  - Journal: **Unsupervised cell functional annotation for single-cell RNA-seq**. Dongshunyi Li, Jun Ding, Ziv Bar-Joseph. *Genome Research*, 32(9): 1765-1775, 2022.
    - DOI: [10.1101/gr.276609.122](https://doi.org/10.1101/gr.276609.122)


- **A novel matrix factorization model for interpreting single-cell gene expression from biologically heterogeneous data**. Kun Qian, Shiwei Fu, Hongwei Li, Wei Vivian Li.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 353-355, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_25](https://doi.org/10.1007/978-3-031-04749-7_25)
  - Preprint: [bioRxiv 2021.10.13.464306 ](https://doi.org/10.1101/2021.10.13.464306 )
  - Journal: **scINSIGHT for interpreting single-cell gene expression from biologically heterogeneous data**. Kun Qian, Shiwei Fu, Hongwei Li, Wei Vivian Li. *Genome Biology*, 23(1): 82 20, 2022.
    - DOI: [10.1186/s13059-022-02649-3](https://doi.org/10.1186/s13059-022-02649-3)


- **Tractable and expressive generative models of genetic variation data**. Meihua Dang, Anji Liu, Xinzhu Wei, Sriram Sankararaman, Guy Van den Broeck.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 356-357, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_26](https://doi.org/10.1007/978-3-031-04749-7_26)
  - Preprint: [bioRxiv 2023.05.16.541036](https://doi.org/10.1101/2023.05.16.541036)


- **CONCERT: Genome-wide prediction of sequence elements that modulate DNA replication timing**. Yang Yang, Yuchuan Wang, Yang Zhang, Jian Ma.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 358-359, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_27](https://doi.org/10.1007/978-3-031-04749-7_27)
  - Preprint: [bioRxiv 2022.04.21.488684](https://doi.org/10.1101/2022.04.21.488684)


- **CORSID enables de novo identification of transcription regulatory sequences and genes in coronaviruses**. Chuanyi Zhang, Palash Sashittal, Mohammed El-Kebir.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 360-362, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_28](https://doi.org/10.1007/978-3-031-04749-7_28)
  - Preprint: [bioRxiv 2021.11.10.468129](https://doi.org/10.1101/2021.11.10.468129)
  - Journal: **Accurate Identification of Transcription Regulatory Sequences and Genes in Coronaviruses**. Chuanyi Zhang, Palash Sashittal, Michael Xiang, Yichi Zhang, Ayesha Kazi, Mohammed El-Kebir. *Molecular Biology and Evolution *, 39(7): msac133, 2022.
    - DOI: [10.1093/molbev/msac133](https://doi.org/10.1093/molbev/msac133)


- **Learning probabilistic protein-DNA recognition codes from DNA-binding specificities using structural mappings**. Joshua Wetzel, Kaiqian Zhang, Mona Singh.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 363-365, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_29](https://doi.org/10.1007/978-3-031-04749-7_29)
  - Preprint: [bioRxiv 2022.01.31.477772](https://doi.org/10.1101/2022.01.31.477772)
  - Journal: **Learning probabilistic protein-DNA recognition codes from DNA-binding specificities using structural mappings**. Joshua L Wetzel, Kaiqian Zhang, Mona Singh . *Genome Research*, 32(9):1776-1786, 2022.
    - DOI: [10.1101/gr.276606.122](https://doi.org/10.1101/gr.276606.122)


- **Uncertainty quantification using subsampling for assembly-free estimates of genomic distance and phylogenetic relationships**. Eleonora Rachtman, Shahab Sarmashghi, Vineet Bafna, Siavash Mirarab.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 366-368, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_30](https://doi.org/10.1007/978-3-031-04749-7_30)
  - Journal: **Uncertainty Quantification Using Subsampling for Assembly-Free Estimates of Genomic Distance and Phylogenetic Relationships**. Eleonora Rachtman, Shahab Sarmashghi, Vineet Bafna, Siavash Mirarab. *Cell Systems*, 13(10): 817-829.e3, 2022.
    - DOI: [10.1016/j.cels.2022.06.007](https://doi.org/10.1016/j.cels.2022.06.007)


- **SOPHIE: viral outbreak investigation and transmission history reconstruction in a joint phylogenetic and network theory framework**. Pavel Skums, Fatemeh Mohebbi, Viachaslau Tsyvina, Pelin Burcak Icer, Yury Khudyakov.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 369-370, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_31](https://doi.org/10.1007/978-3-031-04749-7_31)
  - Preprint: [bioRxiv 2022.05.05.490757](https://doi.org/10.1101/2022.05.05.490757)
  - Journal: **SOPHIE: Viral outbreak investigation and transmission history reconstruction in a joint phylogenetic and network theory framework**. Pavel Skums, Fatemeh Mohebbi, Vyacheslav Tsyvina, Pelin Icer Baykal, Alina Nemira, Sumathi Ramachandran, Yury Khudyakov. *Cell Systems*, 13(10):844-856.e4, 2022.
    - DOI: [10.1016/j.cels.2022.07.005](https://doi.org/10.1016/j.cels.2022.07.005)


- **Identifying systematic variation at the single-cell level by leveraging low-resolution population-level data**. Elior Rahmani, Michael Jordan, Nir Yosef.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 371-371, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_32](https://doi.org/10.1007/978-3-031-04749-7_32)
  - Preprint: [bioRxiv 2022.01.27.478115](https://doi.org/10.1101/2022.01.27.478115)


- **Belayer: Modeling distinct cell type clusters and continuous variation of expression in spatial transcriptomics from layered tissues**. Cong Ma, Uthsav Chitra, Shirley Zhang, Benjamin Raphael.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 372-373, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_33](https://doi.org/10.1007/978-3-031-04749-7_33)
  - Preprint: [bioRxiv 2022.02.05.479261](https://doi.org/10.1101/2022.02.05.479261)
  - Journal: **Belayer: Modeling discrete and continuous spatial variation in gene expression from spatially resolved transcriptomics**. Cong Ma, Uthsav Chitra, Shirley Zhang, Benjamin J Raphael. *Cell Systems*, 13(10): 786-797.e13, 2022.
    - DOI: [10.1016/j.cels.2022.09.002](https://doi.org/10.1016/j.cels.2022.09.002)


- **Lossless Indexing with Counting de Bruijn Graphs**. Mikhail Karasikov, Harun Mustafa, Gunnar Ratsch, Andre Kahles.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 374-376, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_34](https://doi.org/10.1007/978-3-031-04749-7_34)
  - Preprint: [bioRxiv 2021.11.09.467907](https://doi.org/10.1101/2021.11.09.467907)
  - Journal: **Lossless indexing with counting de Bruijn graphs**. Mikhail Karasikov, Harun Mustafa, Gunnar Ratsch, Andre Kahles. *Genome Research*, 32(9):1754-1764, 2022.
    - DOI: [10.1101/gr.276607.122](https://doi.org/10.1101/gr.276607.122)


- **Uncovering hidden assembly artifacts: when unitigs are not safe and bidirected graphs are not helpful**. Amatur Rahman, Paul Medvedev.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 377-379, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_35](https://doi.org/10.1007/978-3-031-04749-7_35)
  - Preprint: [bioRxiv 2022.01.20.477068](https://doi.org/10.1101/2022.01.20.477068)
  - Journal: **Assembler artifacts include misassembly because of unsafe unitigs and underassembly because of bidirected graphs**. Amatur Rahman, Paul Medvedev. *Genome Research*, 32(9): 1746-1753, 2022.
    - DOI: [10.1101/gr.276601.122](https://doi.org/10.1101/gr.276601.122)


- **Mapping single-cell transcriptomes to copy number evolutionary trees**. Pedro F Ferreira, Jack Kuipers, Niko Beerenwinkel.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 380-381, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_36](https://doi.org/10.1007/978-3-031-04749-7_36)
  - Preprint: [bioRxiv 2021.11.04.467244](https://doi.org/10.1101/2021.11.04.467244)


- **ImmunoTyper-SR: A Novel Computational Approach for Genotyping Immunoglobulin Heavy Chain Variable Genes using Short Read Data**. Michael Ford, Ananth Hari, Oscar Rodriguez, Junyan Xu, Justin Lack, Cihan Oguz, Yu Zhang, Sarah Weber, Mary Magliocco, Jason Barnett, Sandhya Xirasagar, Smilee Samuel, Luisa Imberti, Paolo Bonfanti, Andrea Biondi, Clifton Dalgard, Stephen Chanock, Lindsey Rosen, Steven Holland, Helen Su, Luigi Notarangelo, Uzi Vishkin, Corey Watson, S Cenk Sahinalp.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 382-384, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_37](https://doi.org/10.1007/978-3-031-04749-7_37)
  - Preprint: [bioRxiv 2022.01.31.478564](https://doi.org/10.1101/2022.01.31.478564)
  - Journal: **ImmunoTyper-SR: A computational approach for genotyping immunoglobulin heavy chain variable genes using short-read data**. Michael K. B. Ford, Ananth Hari, Oscar Rodriguez, Junyan Xu, Justin Lack, Cihan Oguz, Yu Zhang, Sarah Weber, Mary Magliocco, Jason Barnett, Sandhya Xirasagar, Smilee Samuel, Luisa Imberti, Paolo Bonfanti, Andrea Biondi, Clifton L. Dalgard, Stephen Chanock, Lindsey Rosen, Steven Holland, Helen Su, Luigi Notarangelo,  NIAID COVID Consortium; Uzi Vishkin, Corey T. Watson, S. Cenk Sahinalp. *Cell Systems*, 13(10): 808-816.e5, 2022.
    - DOI: [10.1016/j.cels.2022.08.008](https://doi.org/10.1016/j.cels.2022.08.008)


- **AutoComplete: Deep Learning-based Phenotype Imputation for Large-scale Biomedical Data**. Ulzee An, Na Cai, Andy Dahl, Sriram Sankararaman.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 385-386, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_38](https://doi.org/10.1007/978-3-031-04749-7_38)
  - Preprint: [bioRxiv 2022.08.15.503991](https://doi.org/10.1101/2022.08.15.503991)
  - Journal: **Deep learning-based phenotype imputation on population-scale biobank data increases genetic discoveries**. Ulzee An, Ali Pazokitoroudi, Marcus Alvarez, Lianyun Huang, Silviu Bacanu, Andrew J. Schork, Kenneth Kendler, Päivi Pajukanta, Jonathan Flint, Noah Zaitlen, Na Cai, Andy Dahl. Sriram Sankararaman. *Nature Genetics*, 55(12):2269-2276, 2023.
    - DOI: [10.1038/s41588-023-01558-w](https://doi.org/10.1038/s41588-023-01558-w)


- **Resistor: an algorithm for predicting resistance mutations using Pareto optimization over multistate protein design and mutational signatures**. Nathan Guerin, Teresa Kaserer, Bruce Donald.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 387-389, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_39](https://doi.org/10.1007/978-3-031-04749-7_39)
  - Preprint: [bioRxiv 2022.01.18.476733](https://doi.org/10.1101/2022.01.18.476733)
  - Journal: **Resistor: An algorithm for predicting resistance mutations via Pareto optimization over multistate protein design and mutational signatures**. Nathan Guerin, Andreas Feichtner, Eduard Stefan, Teresa Kaserer, Bruce Donald. *Cell Systems*, 3(10):830-843.e3, 2022.
    - DOI: [10.1016/j.cels.2022.09.003](https://doi.org/10.1016/j.cels.2022.09.003)


- **Ultra high diversity factorizable libraries for efficient therapeutic discovery**. Zheng Dai, Sachit Saksena, Geraldine Horny, Christine Banholzer, Stefan Ewert, David Gifford.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2022. Lecture Notes in Computer Science, vol 13278, pp 390-392, Springer, Cham.
    - DOI: [10.1007/978-3-031-04749-7_40](https://doi.org/10.1007/978-3-031-04749-7_40)
  - Preprint: [bioRxiv 2022.01.17.476670](https://doi.org/10.1101/2022.01.17.476670)
  - Journal: **Ultra-high-diversity factorizable libraries for efficient therapeutic discovery**. Zheng Dai, Sachit D Saksena, Geraldine Horny, Christine Banholzer, Stefan Ewert, David K Gifford. *Genome Research*, 32(9):1787-1794, 2022.
    - DOI: [10.1101/gr.276593.122](https://doi.org/10.1101/gr.276593.122)


