---
title: 2023
nav_order: 3
parent: 2025-2021
---

# RECOMB 2023 Proceedings

İstanbul, Türkiye, April 16-19  
PC Chair: [Haixu Tang](https://homes.luddy.indiana.edu/hatang/), Indiana University  
Organization Committee: Can Alkan (co-chair), Attila Gürsoy (co-chair), Zülal Bingöl, A. Ercüment Çiçek, Tunca Doğan, Ezgi Ebren, Arzucan Özgür, Öznur Taştan  
Keynote Speakers: İvet Bahar, Ewan Birney, Richard Durbin, Tuuli Lappalainen, Sohini Ramachandran, Fabian Theis  

# List of Publications

- **VStrains: de novo reconstruction of viral strains via iterative path extraction from assembly graphs**. Runpeng Luo, Yu Lin.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 3-20, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7_1](https://doi.org/10.1007/978-3-031-29119-7_1)
  - Preprint: [bioRxiv 2022.10.21.513181](https://doi.org/10.1101/2022.10.21.513181)
  
- **Spectrum preserving tilings enable sparse and modular reference indexing**. Jason Fan, Jamshed Khan, Giulio Ermanno Pibiri, Robert Patro.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 21-40, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7_2](https://doi.org/10.1007/978-3-031-29119-7_2)
  - Preprint: [bioRxiv 2022.10.27.513881](https://doi.org/10.1101/2022.10.27.513881)

- **Statistically Consistent Rooting of Species Trees under the Multispecies Coalescent Model**. Yasamin Tabatabaee, Sebastien Roch, Tandy Warnow. 
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 41-57, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7_3](https://doi.org/10.1007/978-3-031-29119-7_3)
  - Preprint: [bioRxiv 2022.10.26.513897](https://doi.org/10.1101/2022.10.26.513897)
  - Journal: **QR-STAR: A Polynomial-Time Statistically Consistent Method for Rooting Species Trees Under the Coalescent**, *Journal of Computational Biology*, 30(11): 1146–1181, 2023
    - DOI: [10.1089/cmb.2023.0185](https://doi.org/10.1089/cmb.2023.0185)
    

- **Sequence to graph alignment using gap-sensitive co-linear chaining**. Ghanshyam Chandra, Chirag Jain.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 58-73, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7_4](https://doi.org/10.1007/978-3-031-29119-7_4)
  - Preprint: [bioRxiv 2022.08.29.505691](https://doi.org/2022.08.29.505691)
  - Journal: **Gap-Sensitive Colinear Chaining Algorithms for Acyclic Pangenome Graphs**. *Journal of Computational Biology*, 30(11): 1182–1197, 2023.
    - DOI: [10.1089/cmb.2023.0186](https://doi.org/10.1089/cmb.2023.0186)


- **DM-Net: a dual-model network for automated biomedical image diagnosis**. Xiaogen Zhou, Zhiqiang Li, Tong Tong.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 74-84, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7_5](https://doi.org/10.1007/978-3-031-29119-7_5)


- **MTGL-ADMET: A Novel Multi-Task Graph Learning Framework for ADMET Prediction Enhanced by Status-Theory and Maximum Flow**. Bing-Xue Du, Yi Xu, Siu Ming Yiu, Hui Yu, Jian-Yu Shi.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 85-103, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7_6](https://doi.org/10.1007/978-3-031-29119-7_6)


- **CDGCN: Conditional de novo Drug generative model using Graph Convolution Networks**. Shikha Mallick, Sahely Bhadra.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 104-119, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7_7](https://doi.org/10.1007/978-3-031-29119-7_7)


- **Percolate: an exponential family JIVE model to design DNA-based predictors of drug response**. Soufiane Mourragui, Marco Loog, Mirrelijn van Nee, Mark van de Wiel, Marcel Reinders, Lodewyk Wessels.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 120-138, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7_8](https://doi.org/10.1007/978-3-031-29119-7_8)
  - Preprint: [bioRxiv 2022.09.11.507473](https://doi.org/2022.09.11.507473)


- **Translation rate prediction and regulatory motif discovery with multi-task learning**. Weizhong Zheng, John H.C. Fong, Yuk Kei Wan, Athena H.Y. Chu, Yuanhua Huang, Alan S.L. Wong, Joshua Ho.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 139-154, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7_9](https://doi.org/10.1007/978-3-031-29119-7_9)
  - Preprint: [bioRxiv 2022.05.03.490410](https://doi.org/2022.05.03.490410)


- **Computing shortest hyperpaths for pathway inference in cellular reaction networks**. Spencer Krieger, John Kececioglu.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 155-173, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7_10](https://doi.org/10.1007/978-3-031-29119-7_10)
  - Journal: **Shortest Hyperpaths in Directed Hypergraphs for Reaction Pathway Inference**. *Journal of Computational Biology*, 30(11): 1198–1225, 2023.
    - DOI: [10.1089/cmb.2023.0242](https://doi.org/10.1089/cmb.2023.0242)


- **T-Cell Receptor Optimization with Reinforcement Learning and Mutation Polices for Precision Immunotherapy**. Ziqi Chen, Martin Min, Hongyu Guo, Chao Cheng, Trevor Clancy, Xia Ning.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 174-192, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7_11](https://doi.org/10.1007/978-3-031-29119-7_11)
  - Preprint: [arXiv:2303.02162](https://doi.org/arXiv:2303.02162)


- **TREE-QMC: Improving quartet graph construction for scalable and accurate species tree estimation from gene trees**. Yunheng Han, Erin K. Molloy.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 195-196, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7](https://doi.org/10.1007/978-3-031-29119-7)
  - Preprint: [bioRxiv 2022.06.25.497608](https://doi.org/2022.06.25.497608)
  - Journal: **Improving quartet graph construction for scalable and accurate species tree estimation from gene trees**. *Genome Research*, 33(7):1042-1052, 2023.
    - DOI: [10.1101/gr.277629.122](https://doi.org/10.1101/gr.277629.122)


- **mapquik: Efficient low-divergence mapping of long reads in minimizer space**. Baris Ekim, Kristoffer Sahlin, Paul Medvedev, Bonnie Berger, Rayan Chikhi.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 197-199, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7](https://doi.org/10.1007/978-3-031-29119-7)
  - Preprint: [bioRxiv 2022.12.23.521809](https://doi.org/2022.12.23.521809)
  - Journal: **Efficient mapping of accurate long reads in minimizer space with mapquik**. *Genome Research*, 33(7): 1188-1197, 2023.
    - DOI: [10.1101/gr.277679.123](https://doi.org/10.1101/gr.277679.123)


- **Deriving confidence intervals for mutation rates across a wide range of evolutionary distances using FracMinHash**. Mahmudur Rahman Hera, N. Tessa Pierce-Ward, David Koslicki.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 200-202, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7](https://doi.org/10.1007/978-3-031-29119-7)
  - Preprint: [bioRxiv 2022.01.11.475870](https://doi.org/2022.01.11.475870)
  - Journal: **Deriving confidence intervals for mutation rates across a wide range of evolutionary distances using FracMinHash**. *Genome Research*, 33(7):1061-1068, 2023.
    - DOI: [10.1101/gr.277651.123](https://doi.org/10.1101/gr.277651.123)


- **Entropy predicts sensitivity of pseudo-random seeds**. Benjamin Dominik Maier, Kristoffer Sahlin.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 203-204, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7](https://doi.org/10.1007/978-3-031-29119-7)
  - Preprint: [bioRxiv 2022.10.13.512198](https://doi.org/2022.10.13.512198)
  - Journal: **Entropy predicts sensitivity of pseudorandom seeds**. *Genome Research*, 33(7): 1162-1174, 2023.
    - DOI: [10.1101/gr.277645.123](https://doi.org/10.1101/gr.277645.123)


- **Seed-chain-extend alignment is accurate and runs in close to O(m log n) time for similar sequences: a rigorous average-case analysis**. Jim Shaw, Yun William Yu.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 205-207, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7](https://doi.org/10.1007/978-3-031-29119-7)
  - Preprint: [bioRxiv 2022.10.14.512303](https://doi.org/2022.10.14.512303)
  - Journal: **Proving sequence aligners can guarantee accuracy in almost O(m log n) time through an average-case analysis of the seed-chain-extend heuristic**. *Genome Research*, 33(7): 1175-1187, 2023.
    - DOI: [10.1101/gr.277637.122](https://doi.org/10.1101/gr.277637.122)


- **Extremely-fast construction and querying of compacted and colored de Bruijn graphs with GGCAT**. Andrea Cracco, Alexandru I. Tomescu.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 208-209, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7](https://doi.org/10.1007/978-3-031-29119-7)
  - Preprint: [bioRxiv 2022.10.24.513174](https://doi.org/2022.10.24.513174)
  - Journal: **Extremely fast construction and querying of compacted and colored de Bruijn graphs with GGCAT**. *Genome Research*, 33(7): 1198-1207, 2023.
    - DOI: [10.1101/gr.277615.122](https://doi.org/10.1101/gr.277615.122)


- **Aligning Distant Sequences to Graphs using Long Seed Sketches**. Amir Joudaki, Alexandru Meterez, Harun Mustafa, Ragnar Groot Koerkamp, André Kahles, Gunnar Rätsch.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 233-235, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7](https://doi.org/10.1007/978-3-031-29119-7)
  - Preprint: [bioRxiv 2022.10.26.513890 ](https://doi.org/2022.10.26.513890 )
  - Journal: **Aligning distant sequences to graphs using long seed sketches**. *Genome Research*, 33(7): 1208-1217, 2023.
    - DOI: [10.1101/gr.277659.123](https://doi.org/10.1101/gr.277659.123)


- **MD-Cat: Phylogenetic Dating under a Flexible Categorical Model using Expectation-Maximization**. Uyen Mai, Eduardo Charvel, Siavash Mirarab.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 236-238, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7](https://doi.org/10.1007/978-3-031-29119-7)
  - Preprint: [bioRxiv 2022.10.06.511147](https://doi.org/2022.10.06.511147)
  - Journal: **Expectation-Maximization enables Phylogenetic Dating under a Categorical Rate Model**. *Systematic Biology*, 73(5): 823-838, 2024.
    - DOI: [10.1093/sysbio/syae034](https://doi.org/10.1093/sysbio/syae034)


- **Phenotypic subtyping via contrastive learning**. Aditya Gorla, Sriram Sankararaman, Esteban Burchard, Jonathan Flint, Noah Zaitlen, Elior Rahmani.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 239-240, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7](https://doi.org/10.1007/978-3-031-29119-7)
  - Preprint: [bioRxiv 2023.01.05.522921 ](https://doi.org/2023.01.05.522921 )


- **HOGVAX: Exploiting Peptide Overlaps to Maximize Population Coverage in Vaccine Design with Application to SARS-CoV-2**. Sara C. Schulte, Alexander Dilthey, Gunnar W. Klau.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 241-243, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7](https://doi.org/10.1007/978-3-031-29119-7)
  - Preprint: [bioRxiv 2023.01.09.523288](https://doi.org/2023.01.09.523288)
  - Journal: **HOGVAX: Exploiting epitope overlaps to maximize population coverage in vaccine design with application to SARS-CoV-2**. *Cell Systems*, 14(12): 1122-1130, 2023.
    - DOI: [10.1016/j.cels.2023.11.001](https://doi.org/10.1016/j.cels.2023.11.001)


- **Ultra-fast genome-wide inference of pairwise coalescence times (Final version)**. Regev Schweiger, Richard Durbin.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 244-246, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7](https://doi.org/10.1007/978-3-031-29119-7)
  - Preprint: [bioRxiv 2023.01.06.522935 ](https://doi.org/2023.01.06.522935 )
  - Journal: **Ultrafast genome-wide inference of pairwise coalescence times**. *Genome Research*, 33(7): 1023-1031, 2023.
    - DOI: [10.1101/gr.277665.123](https://doi.org/10.1101/gr.277665.123)


- **Leveraging family data to design Mendelian Randomization that is provably robust to population stratification**. Nathan LaPierre, Boyang Fu, Steven Turnbull, Eleazar Eskin, Sriram Sankararaman.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 247-248, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7](https://doi.org/10.1007/978-3-031-29119-7)
  - Preprint: [bioRxiv 2023.01.05.522936](https://doi.org/2023.01.05.522936)
  - Journal: **Leveraging family data to design Mendelian randomization that is provably robust to population stratification**. *Genome Research*, 33(7): 1032-1041, 2023.
    - DOI: [10.1101/gr.277664.123](https://doi.org/10.1101/gr.277664.123)


- **Minimal Positional Substring Cover: A Haplotype Threading Alternative to Li & Stephens Model**. Ahsan Sanaullah, Degui Zhi, Shaojie Zhang.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 249-250, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7](https://doi.org/10.1007/978-3-031-29119-7)
  - Preprint: [bioRxiv 2023.01.04.522803 ](https://doi.org/2023.01.04.522803 )
  - Journal: **Minimal positional substring cover is a haplotype threading alternative to Li and Stephens model**. *Genome Research*, 33(7): 1007-1014, 2023.
    - DOI: [10.1101/gr.277673.123](https://doi.org/10.1101/gr.277673.123)


- **Cell segmentation for high-resolution spatial transcriptomics**. Hao Chen, Dongshunyi Li, Ziv Bar-Joseph.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 251-253, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7](https://doi.org/10.1007/978-3-031-29119-7)
  - Preprint: [bioRxiv 2023.01.11.523658](https://doi.org/2023.01.11.523658)
  - Journal: **SCS: cell segmentation for high-resolution spatial transcriptomics**. *Nature Methods*, 20(8): 1237-1243, 2023.
    - DOI: [10.1038/s41592-023-01939-3](https://doi.org/10.1038/s41592-023-01939-3)


- **Unsupervised Deep Peak Caller for ATAC-seq**. Yudi Zhang, Ha Vu, Geetu Tuteja, Karin Dorman.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 254-256, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7](https://doi.org/10.1007/978-3-031-29119-7)
  - Preprint: [bioRxiv 2023.01.07.523108](https://doi.org/2023.01.07.523108)
  - Journal: **Unsupervised contrastive peak caller for ATAC-seq**. *Genome Research*, 33(7): 1133-1144, 2023.
    - DOI: [10.1101/gr.277677.123](https://doi.org/10.1101/gr.277677.123)


- **Unraveling causal gene regulation from the RNA velocity graph using Velorama**. Rohit Singh, Alexander Wu, Anish Mudide, Bonnie Berger.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 257-258, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7](https://doi.org/10.1007/978-3-031-29119-7)
  - Preprint: [bioRxiv 2022.10.18.512766](https://doi.org/2022.10.18.512766)
  - Journal: **Causal gene regulatory analysis with RNA velocity reveals an interplay between slow and fast transcription factors**. *Cell Systems*, 15(5):462-474, 2024.
    - DOI: [10.1016/j.cels.2024.04.005](https://doi.org/10.1016/j.cels.2024.04.005)


- **PIsToN: Evaluating Protein Binding Interfaces with Transformer Networks**. Vitalii Stebliankin, Azam Shirali, Prabin Baral, Prem Chapagain, Giri Narasimhan.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 259-261, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7](https://doi.org/10.1007/978-3-031-29119-7)
  - Preprint: [bioRxiv 2023.01.03.522623](https://doi.org/2023.01.03.522623)
  - Journal: **Evaluating protein binding interfaces with transformer networks**. *Nature Machine Intelligence*, 5: 1042–1053, 2023.
    - DOI: [10.1038/s42256-023-00715-4](https://doi.org/10.1038/s42256-023-00715-4)


- **DebiasedDTA: A Framework for Improving the Generalizability of Drug-Target Affinity Prediction Models**. Rıza Özçelik, Alperen Bağ, Berk Atıl, Melih Barsbey, Arzucan Özgür, Elif Ozkirimli.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 262-264, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7](https://doi.org/10.1007/978-3-031-29119-7)
  - Preprint: [arXiv:2107.05556](https://doi.org/arXiv:2107.05556)
  - Journal: **A Framework for Improving the Generalizability of Drug–Target Affinity Prediction Models**. *Journal of Computational Biology*, 30(11): 1226–1239, 2023.
    - DOI: [10.1089/cmb.2023.0208](https://doi.org/10.1089/cmb.2023.0208)


- **Drug Synergistic Combinations Predictions via Large-Scale Pre-Training and Graph Structure Learning**. Zhihang Hu, Qinze Yu, Yucheng Guo, Taifeng Wang, Irwin King, Xin Gao, Le Song, Yu Li.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 265-267, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7](https://doi.org/10.1007/978-3-031-29119-7)
  - Preprint: [arXiv:2301.05931](https://doi.org/arXiv:2301.05931)


- **Pisces: a cross-modal contrastive learning approach to synergistic drug combination prediction**. Jiacheng Lin, Hanwen Xu, Addie Woicik, Jianzhu Ma, Sheng Wang.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 268-270, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7](https://doi.org/10.1007/978-3-031-29119-7)
  - Preprint: [bioRxiv 2022.11.21.517439](https://doi.org/2022.11.21.517439)
  - Journal: **Pisces: A multi-modal data augmentation approach for drug combination synergy prediction**. *Cell Genomics*, 5(7): 100892, 2025.
    - DOI: [10.1016/j.xgen.2025.100892](https://doi.org/10.1016/j.xgen.2025.100892)


- **Modeling and Predicting Cancer Clonal Evolution with Reinforcement Learning**. Stefan Ivanovic, Mohammed El-Kebir.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 271-273, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7](https://doi.org/10.1007/978-3-031-29119-7)
  - Preprint: [bioRxiv 2022.12.11.519917](https://doi.org/2022.12.11.519917)
  - Journal: **Modeling and predicting cancer clonal evolution with reinforcement learning**. *Genome Research*, 33(7): 1078-1088, 2023.
    - DOI: [10.1101/gr.277672.123](https://doi.org/10.1101/gr.277672.123)


- **Enabling Trade-offs in Privacy and Utility in Genomic Data Beacons and Summary Statistics**. Rajagopal Venkatesaramani, Zhiyu Wan, Brad Malin, Yevgeniy Vorobeychik.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 274-276, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7](https://doi.org/10.1007/978-3-031-29119-7)
  - Preprint: [arXiv:2302.01763](https://doi.org/arXiv:2302.01763)
  - Journal: **Enabling tradeoffs in privacy and utility in genomic data Beacons and summary statistics**. *Genome Research*, 33(7): 1113-1123, 2023.
    - DOI: [10.1101/gr.277674.123](https://doi.org/10.1101/gr.277674.123)


- **Accurate evaluation of transcriptomic re-identification risks using discriminative sequence models**. Shuvom Sadhuka, Daniel Fridman, Bonnie Berger, Hyunghoon Cho.
  - Proceedings: Research in Computational Molecular Biology. RECOMB 2023. Lecture Notes in Computer Science, vol 13976, pp 277-279, Springer, Cham.
    - DOI: [10.1007/978-3-031-29119-7](https://doi.org/10.1007/978-3-031-29119-7)
  - Preprint: [bioRxiv 2023.04.13.536784](https://doi.org/2023.04.13.536784)
  - Journal: **Assessing transcriptomic reidentification risks using discriminative sequence models**. *Genome Research*, 33(7):1101–1112, 2023.
    - DOI: [10.1101/gr.277699.123](https://doi.org/10.1101/gr.277699.123)


